forge-gui/res/cardsfolder/upcoming/glorfindel_dauntless_rescuer.txt

```diff
@@ -0,0 +1,11 @@
+Name:Glorfindel, Dauntless Rescuer
+ManaCost:2 G
+Types:Legendary Creature Elf Noble
+PT:3/2
+T:Mode$ Scry | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCharm | TriggerDescription$ Whenever you scry, choose one and CARDNAME gets +1/+1 until end of turn.
+SVar:TrigCharm:DB$ Charm | Choices$ DBMustBeBlocked,DBEffect | SubAbility$ DBPump | SpellDescription$ 
+SVar:DBMustBeBlocked:DB$ Pump | Defined$ Self | KW$ HIDDEN CARDNAME must be blocked if able. | SpellDescription$ NICKNAME must be blocked this turn if able.
+SVar:DBEffect:DB$ Effect | StaticAbilities$ MinMaxBlocked | SpellDescription$ NICKNAME can't be blocked by more than one creature each combat this turn.
+SVar:MinMaxBlocked:Mode$ MinMaxBlocker | ValidCard$ Creature.Self | Max$ 1 | Description$ NICKNAME can't be blocked by more than one creature each combat this turn.
```

Comment:
that Valid shouldn't work?

Reply:
still a bit wrong
(that's Defined syntax but like the param name implies it only accepts Valid)